plugins/openquakeqgiswebservices/openquakeqgiswebservices.py

```diff
@@ -0,0 +1,65 @@
+import json
+from qgis.server import QgsService
+
+
+class OpenQuakeQGISWebServices(QgsService):
+
+    def __init__(self):
+        QgsService.__init__(self)
+
+    def name(self):
+        return "OQWS"
+
+    def version(self):
+        return "1.0.0"
+
+    def allowMethod(method):
+        return True
+
+    def executeRequest(self, request, response, project):
+        if request.parameter('REQUEST') == 'GetCustomPropertiesByLayerName':
+            service = self._get_custom_properties_by_layer_name
+        elif request.parameter('REQUEST') == 'GetCustomPropertiesByLayerId':
```

Comment:
Instead of using two different `Get*` we may rely on the corresponding project setting, like `WMS` does:

![Screenshot from 2019-03-18 15-47-52](https://user-images.githubusercontent.com/1818657/54538877-60882280-4995-11e9-8568-4da83d00d269.png)

So, based on this setting we can use `LayerName` or `LayerId` as key the dictionary (and to filter based on the `LAYERS` parameter.

Reply:
I still haven't had time to find that property from within the plugin. So far, I've found how it is set inside the project xml:
```bash
diff checked.qgs unchecked.qgs
3292c3292
<     <WMSUseLayerIDs type="bool">true</WMSUseLayerIDs>
---
>     <WMSUseLayerIDs type="bool">false</WMSUseLayerIDs>
```